### PR TITLE
Operational DS controller handlers

### PR DESCRIPTION
**Pull request by tomsou**
operation Datastore handlers and methods implemented (oo approach)

### CODE REVIEW

File: controllers/odl_beryllium_pb/get_flows.py [hunk 1]
```diff
@@ -1,8 +1,10 @@
 #! /usr/bin/env python3.4
```
Comment (anastop) [inline controllers/odl_beryllium_pb/get_flows.py]: add a module docstring to describe what the module does, plus the cmd line positional arguments

do this for every handler

File: stress_test/controller.py [hunk 2]
```diff
@@ -54,45 +56,41 @@ def __init__(self, ctrl_base_dir, test_config):
 
         self._ssh_conn = None
 
-        #check handlers' validity
+        '''check handlers' validity'''
```
Comment (anastop) [inline stress_test/controller.py]: Don't use triple backticks for inline or block comments 

The convention is (https://www.python.org/dev/peps/pep-0008/#id29):
- use triple backticks (`'''`) for docstring comments only (at module level, class level, function level)
- use `#` for inline comments or block comments

Change this everywhere else you might have it

File: stress_test/controller.py [hunk 3]
```diff
@@ -41,11 +38,16 @@ def __init__(self, ctrl_base_dir, test_config):
         self.of_port = test_config['controller_port']
         self.logs_dir = self.base_dir + test_config['controller_logs_dir']
 
-        self.build_hnd = self.base_dir + test_config['controller_build_handler']
-        self.start_hnd = self.base_dir + test_config['controller_start_handler']
-        self.stop_hnd = self.base_dir + test_config['controller_stop_handler']
-        self.status_hnd = self.base_dir + test_config['controller_status_handler']
-        self.clean_hnd = self.base_dir + test_config['controller_clean_handler']
+        self.build_hnd = self.base_dir + test_config['controller_'
```
Comment (anastop) [inline stress_test/controller.py]: Do not split a single string, it worsens a lot code readability. 

There are two choices here:
1. either leave the whole statement as is, without further splitting it
2. either split the line at some operator, e.g. `+`

I would prefer option 1, even if it violates the line-length limits

Check this everywhere

File: stress_test/controller_test_basic.py [hunk 4]
```diff
@@ -11,100 +11,142 @@
...
+        logging.info("[Testing] Interval statistics not updated")
+
+    '''get information about Controller Datastore'''
+    try:
+        hosts = ctrl.get_oper_hosts()
+        print('[Testing] The number of hosts on datastore '
+              'are: {0}'.format(hosts))
+    except AttributeError:
+        print('[Testing] Error during the query of hosts in DS')
+    try:
```
Comment (anastop) [inline stress_test/controller_test_basic.py]: leave an empty line between consecutive try-except blocks, in order to separate them

do this everywhere

File: stress_test/controller_test_basic.py [hunk 5]
```diff
@@ -11,100 +11,142 @@
...
+                                                       cmd,
+                                                       'Check_stats_period')
+    if (output is not None):
+        logging.info('[Testing] Interval statistics have been '
+                     'updated successfully')
     else:
-        logging.info ("[Testing] Interval statistics not updated")
-
+        logging.info("[Testing] Interval statistics not updated")
+
```
Comment (anastop) [inline stress_test/controller_test_basic.py]: have you checked this against a running topology ? (e.g. Mininet)

File: stress_test/controller.py (hunk 3 above)
Comment (tomsou) [inline stress_test/controller.py]: Updated, adopting the 1st option

File: stress_test/controller_test_basic.py (hunk 5 above)
Comment (tomsou) [inline stress_test/controller_test_basic.py]: yes, it is has been tested by the means of the controller_test_basic.py on a Mininet topology with 5 switches, 5 hosts, 10 links and 13 flows

File: stress_test/controller.py (hunk 2 above)
Comment (tomsou) [inline stress_test/controller.py]: done, in controller.py and controller_test_basic_py files

File: stress_test/controller_test_basic.py (hunk 4 above)
Comment (tomsou) [inline stress_test/controller_test_basic.py]: done

File: stress_test/controller_test_basic.py (hunk 5 above)
Comment (anastop) [inline stress_test/controller_test_basic.py]: great!

File: stress_test/controller.py (hunk 3 above)
Comment (anastop) [inline stress_test/controller.py]: great

File: controllers/odl_beryllium_pb/get_flows.py [hunk 6]
```diff
@@ -1,8 +1,16 @@
...
-import emulators.nb_generator.flow_utils
-import logging
+
+"""This handler returns the number of installed flows of a topology, connected
+to the controller. This information is extracted from controller's operational
+datastore, using RestCall.
```
Comment (anastop) [inline controllers/odl_beryllium_pb/get_flows.py]: have you seen this comment?

File: controllers/odl_beryllium_pb/controller_flowmods_configure.py [hunk 7]
```diff
@@ -1,10 +1,16 @@
 #! /usr/bin/env python3.4
```
Comment (anastop) [inline controllers/odl_beryllium_pb/controller_flowmods_configure.py]: have you seen this comment?

Comment (tomsou) [inline controllers/odl_beryllium_pb/controller_flowmods_configure.py]: done

File: controllers/odl_beryllium_pb/get_flows.py (hunk 6 above)
Comment (tomsou) [inline controllers/odl_beryllium_pb/get_flows.py]: done, for all get_xxx.py files

File: controllers/odl_beryllium_pb/change_persistence.py [hunk 8]
```diff
@@ -1,12 +1,19 @@
 #! /usr/bin/env python3.4
 
+"""This handler disables the percistence mode in the configuration of the
```
Comment (tomsou) [inline controllers/odl_beryllium_pb/change_persistence.py]: done

**Pull request merged**